lib/nolibc/string.c

```diff
@@ -799,6 +817,18 @@ char *strerror(int errnum)
 	return strerror_r(errnum, buf, sizeof(buf));
 }
 
+char *strcat(char *restrict dest, const char *restrict src)
+{
+	char *out = dest;
+
+	out = out + strlen(dest);
+	while (*src != 0)
+		*out++ = *src++;
+
+	*out = 0;
+	return dest;
+}
```

Comment:
In musl this is implemented as

```C
strcpy(dest + strlen(dest), src);
```
I was just wondering if you are gaining any performance benefit from one or the other. If we would implement an optimized version of strcpy for example, we would benefit here also. But maybe until then, we have an additional function call 🤔